docs/topics/downloader-middleware.rst

```diff
@@ -983,7 +980,7 @@ RobotsTxtMiddleware
     Scrapy ships with support for the following robots.txt_ parsers:
 
     * :ref:`Protego <protego-parser>` (default)
-    * :ref:`RobotFileParser <python-robotfileparser>`
+    * :class:`~urllib.robotparser.RobotFileParser`
```

Comment:
I think this line should stay _as is_, and the `python-robotfileparser` reference marker below restored as well.